### ISSUE TITLE
Windows build: don't include make in installer

### ISSUE THREAD

**Issue by MSoegtropIMC** [coq/coq#7172]
Currently the Windows build includes a MinGW make, but this make leads to problems (because it doesn't handle posix shell commands). Since it hides other make in the system, it is better to remove it by default.

there is a top level script option -make. Things to change are:

- set the default to N
- remove the override if MAKEINSTALLER==Y

**Comment by Zimmi48** [comment]
Isn't this #5987 which was fixed in 951f80b0ce46976791d8b7fa7cf68aea11018da0?

**Comment by MSoegtropIMC** [comment]
I don't think it actually fixes it - my bad, because I suggested this solution.

There is the following code in the top level bat file MakeCoq_MinGW.bat:

IF "%MAKEINSTALLER%" == "Y" (
  SET INSTALLMODE=relocatable
  SET INSTALLOCAML=Y
  SET INSTALLMAKE=Y
)

so MAKEINSTALLER=Y overrides INSTALLMAKE to Y, so in the end installers will always have make regardless of the -make setting.

Besides the changes done in #5987 I also wanted to change the default setting - not just the call in CI.

Sorry, I should have referenced #5987.

**Comment by MSoegtropIMC** [comment]
@gares : in case this is urgent (for 8.8 or so) please set the priority accordingly.

**Comment by MSoegtropIMC** [comment]
Ok, will do it today.
